packages/@aws-cdk/aws-apigatewayv2-authorizers/lib/http/user-pool.ts

```diff
@@ -48,8 +47,7 @@ export class HttpUserPoolAuthorizer implements IHttpRouteAuthorizer {
 
   public bind(options: HttpRouteAuthorizerBindOptions): HttpRouteAuthorizerConfig {
     if (!this.authorizer) {
-      const id = this.props.authorizerName && !Token.isUnresolved(this.props.authorizerName) ?
```

Comment:
Must still check for tokens!

Reply:
Since this id is used as the construct's id (the `HttpAuthorizer` construct), there is already a check in the CDK that disallows construct ids to be tokens.

Check what error message we get when this happens. if the error message is ambiguous, it does not hurt to add a check here with a clear message 'authorizerName must not be a token'